refactor_layout.py

```diff
@@ -0,0 +1,1633 @@
+                ("frc/motorcontrol/PWMVictorSPX.h", "wpi/hardware/motor/PWMVictorSPX" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/Jaguar.h", "wpi/hardware/motor/Jaguar" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/Victor.h", "wpi/hardware/motor/Victor" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/DMC60.h", "wpi/hardware/motor/DMC60" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/VictorSP.h", "wpi/hardware/motor/VictorSP" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/PWMVenom.h", "wpi/hardware/motor/PWMVenom" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/Koors40.h", "wpi/hardware/motor/Koors40" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/SD540.h", "wpi/hardware/motor/SD540" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/Talon.h", "wpi/hardware/motor/Talon" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/Spark.h", "wpi/hardware/motor/Spark" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/PWMTalonSRX.h", "wpi/hardware/motor/PWMTalonSRX" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/PWMTalonFX.h", "wpi/hardware/motor/PWMTalonFX" + NEW_CC_FILE_SUFFIX),
+                ("frc/motorcontrol/PWMSparkMax.h", "wpi/hardware/motor/PWMSparkMax" + NEW_CC_FILE_SUFFIX),
+            ],
+        ),
+    ]
+
+    # Package changes (and therefor folder moves) for java projects.
+    # Format: (project, original package, new package)
+    JAVA_PROJECT_REPLACMENTS = [
+        ("wpiutil", "edu.wpi.first.util", "org.wpilib.util"),
+        ("wpimath", "edu.wpi.first.math", "org.wpilib.math"),
+        ("wpinet", "edu.wpi.first.net", "org.wpilib.net"),
+        ("datalog", "edu.wpi.first.datalog", "org.wpilib.datalog"),
+        ("command", "edu.wpi.first.wpilibj2.command", "org.wpilib.command"),
+        ("ntcore", "edu.wpi.first.networktables", "org.wpilib.networktables"),
+        ("fields", "edu.wpi.first.fields", "org.wpilib.fields"),
+        ("hal", "edu.wpi.first.hal", "org.wpilib.hardware.hal"),
+        ("wpiunits", "edu.wpi.first.units", "org.wpilib.units"),
+        ("apriltag", "edu.wpi.first.apriltag", "org.wpilib.vision.apriltag"),
+        ("epilogue-processor", "edu.wpi.first.epilogue.processor", "org.wpilib.epilogue.processor"),
+        ("epilogue-runtime", "edu.wpi.first.epilogue", "org.wpilib.epilogue"),
+        ("epilogue-runtime", "edu.wpi.first.epilogue.logging", "org.wpilib.epilogue.logging"),
+        ("cscore", "edu.wpi.first.cscore", "org.wpilib.vision.camera"),
+        ("cameraserver", "edu.wpi.first.vision", "org.wpilib.vision.process"),
+        ("cameraserver", "edu.wpi.first.cameraserver", "org.wpilib.vision.stream"),
+        ("xrpVendordep", "edu.wpi.first.wpilibj.xrp", "org.wpilib.xrp"),
+        ("romiVendordep", "edu.wpi.first.wpilibj.romi", "org.wpilib.romi"),
+        ("wpilibjExamples", "edu.wpi.first.wpilibj.commands", "org.wpilib.commands"),
```

Comment:
Maybe `org.wpilib.snippets.commands2`? It should have `2` in the package name, but also shouldn't use `org.wpilib.commands2` since that would split the package which Java does _not_ like.

The files in this package are also fairly useless; personally, I'd suggest just deleting it, but that'd out of scope for this PR